src/Monolog/Handler/LogdnaHandler.php

```diff
@@ -128,4 +143,11 @@ protected function getDefaultFormatter(): FormatterInterface
     {
         return new \Zwijn\Monolog\Formatter\LogdnaFormatter();
     }
+
+    private function getCurrentTimeInMilliseconds(): string
```

Comment:
I would just remove this function or replace it's content with 
```
return (string)floor(microtime(true) * 1000);
```

Reply:
done